Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageSendButton_Tests.swift

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import SwiftUI
+import XCTest
+
+class ChatMessageSendButton_Tests: XCTestCase {
+    private lazy var container = UIView().withoutAutoresizingMaskConstraints
+
+    override func setUp() {
+        super.setUp()
+        container.subviews.forEach { $0.removeFromSuperview() }
+    }
+
+    func test_defaultAppearance_newMessage() {
+        let view = ChatMessageSendButton().withoutAutoresizingMaskConstraints
+        container.embed(view)
+
+        view.isEnabled = true
+        AssertSnapshot(container, variants: .onlyUserInterfaceStyles, suffix: "new-enabled")
+
+        view.isEnabled = false
+        AssertSnapshot(container, variants: .onlyUserInterfaceStyles, suffix: "new-disabled")
+    }
+
+    func test_appearanceCustomization_usingUIConfig() {
+        var config = UIConfig()
+        config.images.messageComposerSendMessage = UIImage(
+            named: "reaction_love_big",
+            in: .streamChatUI
+        )!
+            .tinted(with: .systemPink)!
```

Comment:
We are trying to use all test images in `TestImages` so they are consistent and don't change whenever we changed them in `StreamChatUI`. Since if that happens we would need to re-record the snapshots